src/client/containers/OrderPage/OrderPage.Container.js

```diff
@@ -1,10 +1,108 @@
+import { useParams, useHistory } from 'react-router-dom';
+import { useFetchApi } from '../../hooks/UseFetchApi';
 import './OrderPage.Style.css';
 
 const OrderPageContainer = () => {
+  const [items, setItems] = React.useState([]);
+  const [order, setOrder] = React.useState({});
+  const [user, setUser] = React.useState({});
+  const [total, setTotal] = React.useState(0);
+  const { id } = useParams();
+  const history = useHistory();
+
+  const newItems = useFetchApi(`orders/${id}`);
+
+  React.useEffect(() => {
+    if (!newItems.isLoading) {
+      setItems(newItems.data.items);
+      setOrder(newItems.data.order);
+    }
+  }, [newItems]);
+
+  const userInfo = useFetchApi(`users/${order.userId}`);
+
+  React.useEffect(() => {
+    if (!userInfo.isLoading) {
+      setUser(userInfo.data[0]);
+    }
+  }, [userInfo, user]);
+
+  React.useEffect(() => {
+    let cost = 0;
+    const subTotal = items.map((item) => item.price * item.quantity);
+
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < subTotal.length; i++) {
+      cost += subTotal[i];
+      console.log(subTotal[i]);
+    }
+    setTotal(cost);
+  }, [items]);
+  console.log(total);
   return (
     <div>
-      <h1>Order page Container</h1>
+      <h1>ORDER SUMMARY</h1>
+      <div className="order-page-container">
+        <div className="top">
+          <div className="left">
+            <div className="orderInfo">
+              <div className="orderId">ORDER ID: {id}</div>
+              <div className="orderStatus">
+                ORDER STATUS: {order.orderStatus}
+              </div>
+            </div>
+            <div className="product">
+              {newItems.isLoading ? (
+                <Loader />
+              ) : (
+                items.map((product) => (
+                  <li key={product.productId}>
+                    <ShoppingItem
+                      productName={product.name}
+                      productImg={product.picture}
+                      quantity={product.stock_amount}
+                      price={product.price}
+                      initValue={product.quantity}
+                      isDisable={true}
+                    />
+                  </li>
+                ))
+              )}
+            </div>
+          </div>
+          <div className="right">
+            <div className="total">
+              <TotalPrice subTotal={total} />
+            </div>
+            <div className="payment-btn">
+              <ButtonComponent
+                title="PAYMENT"
+                backgroundColor="blueviolet"
+                onClick={() => history.push(`paymentSystem/${id}`)}
+              />
+            </div>
+          </div>
+        </div>
+        <div className="botton">
+          <div className="delivery">
+            {userInfo.isLoading ? (
+              <Loader />
+            ) : (
+              <DeliveryInfo editMode={true} vertDisplay={false} user={user} />
```

Comment:
I think you should change this values. If you make like this it will be same with Figma
```suggestion
              <DeliveryInfo editMode={false} vertDisplay={true} user={user} />
```